acousticbrainz/views/data.py

```diff
@@ -44,21 +46,30 @@ def view_high_level(mbid):
 
 @data_bp.route("/<uuid:mbid>", methods=["GET"])
 def summary(mbid):
-    lowlevel, highlevel, genres, moods, other = get_summary_data(mbid)
-
-    info = _get_track_info(mbid, lowlevel['metadata'])
-
-    # Tomahawk player stuff
-    if not ('artist' in lowlevel['metadata']['tags'] and 'title' in lowlevel['metadata']['tags']):
-        tomahawk_url = None
+    try:
+        summary = get_summary_data(mbid)
```

Comment:
Name of this variable is the same as the function. I'm not sure if it's a good idea to do that.